### ISSUE TITLE
Revise BAO terms in EFO

### ISSUE THREAD

**Issue by paolaroncaglia** [EBISPOT/efo#903]
Our current strategy is not to import BAO terms in EFO, but rather cross-reference them.
These 2 BAO terms in EFO should be replaced by CHEBI entries (the BAO IDs can stay as dbxrefs):
BAO:0000925 Glucose-6-phosphate
BAO:0000875 Cyclic adenosine monophosphate
They're live (not obsolete) in BAO (they are placed in the assay biology/method component hierarchy).
They're in the 'chemical entity' hierarchy in EFO.
What would be the correct CHEBI IDs to replace them with?

BAO:0000925 Glucose-6-phosphate:
Wikipedia says that glucose 6-phosphate has IUPAC name D-Glucopyranose 6-phosphate and matches CHEBI:4170 D-glucopyranose 6-phosphate (ChEBI has no generic "glucose 6-phosphate", only D-glucose 6-phosphate which is a parent of CHEBI:4170). The BAO term has no definition or useful annotations/axioms, not even in BAO itself, and is not in Ontobee, so I have no way of knowing how it's used or of being 100% sure that it refers to CHEBI:4170, but it seems reasonable. So I'd suggest to
- [ ] Import CHEBI:4170 D-glucopyranose 6-phosphate (but note, as of today 8/3/22, EFO doesn't import ChEBI dynamically) 
- [ ] Obsolete BAO:0000925 Glucose-6-phosphate, following instructions for Case b) [here](https://docs.google.com/document/d/1bcepkb_nRRQ8LBarc3TnkAdYom5pQLX5VOApwfsxMuA/edit#heading=h.w8upbpwslexz), and indicate CHEBI:4170 as replacement term.

BAO:0000875 Cyclic adenosine monophosphate:
no definition or useful annotations/axioms, not even in BAO itself (except for "has role some (second messenger or measured entity)), and is not in Ontobee, so I have no way of knowing how it's used.
Note that EFO has 'cyclic adenosine monophosphate measurement', gwas=true, but it's not linked to a chemical term (yet). I'd suggest to 
- [ ] Obsolete BAO:0000875 Cyclic adenosine monophosphate, following instructions for Case b) [here](https://docs.google.com/document/d/1bcepkb_nRRQ8LBarc3TnkAdYom5pQLX5VOApwfsxMuA/edit#heading=h.w8upbpwslexz), and indicate [CHEBI:17489 3',5'-cyclic AMP](https://www.ebi.ac.uk/ols/ontologies/chebi/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCHEBI_17489) as replacement term (it's already in EFO so no need to import it).

- [ ] Check if there are any other non-CHEBI chemical terms in the 'chemical entity' branch in EFO that may need replacing.

All of this can probably be put on the back burner until a decision is made about [importing CHEBI dynamically into EFO](https://github.com/EBISPOT/efo/issues/623#issuecomment-1035098325) or not. I'll remove my assignment for now, but feel free to ping me if I'm still here. Thanks.

**Comment by jamesamcl** [comment]
**Status triage (2026-08-26):** still valid and now unblocked — nothing here has been actioned yet.

**The ask:** replace the two BAO chemical terms in EFO's `chemical entity` branch with ChEBI equivalents (keeping the BAO IDs as dbxrefs), then sweep the branch for other non-ChEBI chemical terms.

**Current state on `master`:**
- Both flagged terms are **still present and active** in `src/ontology/efo-edit.owl` (lines ~102051–102066), natively embedded under `http://www.bioassayontology.org/bao#…`:
  - `BAO:0000875` Cyclic adenosine monophosphate
  - `BAO:0000925` Glucose-6-phosphate

  Each has only an `rdfs:label` and `subClassOf CHEBI:24431` — no definition, no xref, no synonyms, not deprecated.
- `om ogrep` shows **no children and no other referrers** for either term, so obsoleting them is self-contained.
- EFO still does **not** import BAO (no `iri_dependencies/bao_terms.txt`, no BAO product in `owlmake.yaml`), consistent with the cross-reference-only policy stated here.
- **The stated blocker is resolved.** ChEBI is now imported dynamically (`chebi` product in `owlmake.yaml`, driven by `iri_dependencies/chebi_terms.txt`, currently 2,097 IRIs) and #623 is closed.
- `CHEBI:17489` 3',5'-cyclic AMP is **already imported** — the replacement for `BAO:0000875` is ready to use.
- `CHEBI:4170` D-glucopyranose 6-phosphate is **not yet in EFO** — one line in `chebi_terms.txt` plus an import rebuild.
- There is precedent for the proposed approach: `efo-edit.owl` already obsoletes foreign-namespace classes in place (BTO ×13, DOID ×6, CL ×5, HP ×2, MP ×1, NCBITaxon ×1).

**On the third checkbox** — still open, and slightly larger than two terms. Foreign-namespace classes currently declared directly under `CHEBI:24431`:
- NCIT ×5 — S-Adenosylhomocysteine, Tilarginine, Guanosine Diphosphate, Gentisic Acid, Alpha-Ketoglutaric Acid, Hippuric Acid
- OMIT ×3 — Adenosine Diphosphate, Guanosine Monophosphate, Uridine Diphosphate
- BAO ×2 — the two above

All look ChEBI-replaceable by the same recipe.

**Also still true:** `EFO:0010473` cyclic adenosine monophosphate measurement has no link to a chemical entity (no `is_about`), as noted in the description.

**Recommendation:** keep open and action the two BAO terms — the work is small, isolated and unblocked. Suggested split:
1. This issue: import `CHEBI:4170`, obsolete `BAO:0000925` → `CHEBI:4170` and `BAO:0000875` → `CHEBI:17489`, retaining the BAO IDs as dbxrefs.
2. A follow-up issue for the third checkbox: the remaining 8 NCIT/OMIT chemical terms under `chemical entity`.

One point worth a maintainer's confirmation first: the reporter was not fully certain that `BAO:0000925` "Glucose-6-phosphate" maps to `CHEBI:4170` (D-glucopyranose 6-phosphate) rather than to its broader parent D-glucose 6-phosphate. The BAO term still carries no definition upstream, so that mapping is a judgement call.

---
*This is an automated triage review by an AI agent; maintainers, please correct if anything here is wrong.*
